game.py

```diff
@@ -0,0 +1,141 @@
+import random
+import string
+
+
+class Game:
+    def __init__(self, size=3, characters=string.ascii_letters, target=100):
+        self.size = size
+        self.characters = characters[:self.size - 1]
+        self.target = target
+        self.board = [[None for _ in range(size)] for _ in range(size)]
```

Comment:
not relevant here, but we can use [None]*size for the first loop.
Do not do that for second loop though :)

Reply:
Thanks, implemented!